README.md

```diff
@@ -62,14 +62,14 @@ Create a new Volto project by using the `@plone/generator-volto` utility.
 
 It will bootstrap a Volto project in a folder of your choice with all the required
 boilerplate to start customizing your Volto site.
-
-    $ npm install -g yo @plone/generator-volto
-    $ yo @plone/volto
-
+```
```

Comment:
```suggestion
```shell
```

Reply:
For readability by editors, I would also suggest a blank line before and after the triple backticks for each code block.

Reply:
BTW, I agree that `$` should be removed from code blocks. Nowadays the tooling works better without the shell prompt. It also differs across various shells.